package.json

```diff
@@ -22,11 +22,11 @@
   "types": "src/node-capnp/capnp.d.ts",
   "scripts": {
     "install": "node ./build.js --dist-url=https://electronjs.org/headers --target=20.3.3",
-    "test": "electron src/node-capnp/capnp-test.js",
-    "build:darwin": "npm run install -- --target_plat=darwin --patch_path=.libs"
```

Comment:
I don't see anything that causes the new toolchain to be run with the replacement for `patch_path` (`patch-lib-path`) which will break the darwin build.

Reply:
Good catch - solved this by hardcoding the `patchLibPath`, since the build paths themselves are currently hardcoded anyway.